packages/payment/src/models/payment.ts

```diff
@@ -63,6 +63,9 @@ export default class Payment {
   @Property({ columnType: "text", nullable: true })
   order_id: string | null
 
+  @Property({ columnType: "text", nullable: true })
+  order_edit_id: string | null
```

Comment:
```suggestion
  order_edit_id?: string | null = null
```

+ add to optional props, waiting discussion output first

cc @oliverheaps

Reply:
if we add that as a convention, should be changed everywhere. But in my opinion it would make things easier to manipulate